TCMSApp/src/client/app/tests/tests.controller.js

```diff
@@ -10,7 +10,13 @@
     function TestsController(logger, FakeTestsFactory, $uibModal) {
         var vm = this;
 
+        vm.stepList = [1,2];
+        vm.addStep = addStep;
+        vm.delStep = delStep;
+
         vm.trigger = 0; // suite -> testCases
```

Comment:
Anyway, I'm working on it :)
We can have currentSuite, but I think it brings more complexity. 
However, it can be handy in future :)